mmdet3d/datasets/pipelines/transforms_3d.py

```diff
@@ -818,45 +818,60 @@ def __repr__(self):
 
 
 @PIPELINES.register_module()
-class IndoorPointSample(object):
-    """Indoor point sample.
+class PointSample(object):
+    """Point sample.
 
     Sampling data to a certain number.
 
     Args:
-        name (str): Name of the dataset.
         num_points (int): Number of points to be sampled.
+        sample_range (float, optional): The range where to sample points.
     """
 
-    def __init__(self, num_points):
+    def __init__(self, num_points, sample_range=None):
         self.num_points = num_points
-
-    def points_random_sampling(self,
-                               points,
-                               num_samples,
-                               replace=None,
-                               return_choices=False):
+        self.sample_range = sample_range
+
+    def _points_random_sampling(self,
+                                points,
+                                num_samples,
+                                sample_range=None,
+                                replace=None,
```

Comment:
1. by default, it should be False? 
2. It seems that it cannot be set from the config/module and it is not used in __call__, so should we keep it?

Reply:
I think `replace` is useful especially when in case that users want to use replace explicitly. So I create an interface for `replace`.

Reply:
Besides, thanks for your remind, I found that most of `replace` in `mmdet3d/datasets/pipelines/transforms_3d.py` are not used in **call**. So there is a choice: keeping shutdown or open for all `replace`.